backend/src/services/bambora.js

```diff
@@ -22,11 +22,16 @@ module.exports = {
     },
 
     createBamboraPaymentRequest: (paymentModel) => {
+        const amount = paymentModel.amount * 100;
+        const preTaxAmount = Math.round(Number((amount * 10 / 11).toFixed(2)));
+        const taxAmount = Math.round(
+            Number((amount - preTaxAmount).toFixed(2))
```

Comment:
Same here, is this conversion necessary?

Reply:
This is a temporary price calculation logic. will be replaced later when the proper prices are available from grynos.